### PR TITLE
Santander AIS

### PR DESCRIPTION
**Pull request by n0rdy**

### CODE REVIEW

File: adapters/santander-adapter/pom.xml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                            <rules>
+                                <bannedDependencies>
+                                    <excludes>
+                                        <!-- project dependencies -->
+                                        <exclude>de.adorsys.xs2a.adapter:xs2a-adapter-app</exclude>
+                                        <exclude>de.adorsys.xs2a.adapter:xs2a-adapter-rest-server</exclude>
+                                        <exclude>de.adorsys.xs2a.adapter:xs2a-adapter-service-loader</exclude>
+                                        <!-- adapters -->
+                                        <exclude>de.adorsys.xs2a.adapter:deutsche-bank-adapter</exclude>
+                                        <exclude>de.adorsys.xs2a.adapter:adorsys-integ-adapter</exclude>
```
Comment (spetrychenko) [inline adapters/santander-adapter/pom.xml]: please add all adapters to the exclusion list. and santander-adapter to other adapters poms. we should think about better way for that

File: xs2a-adapter-service-api/src/main/resources/adapter.config.properties [hunk 2]
```diff
@@ -18,3 +18,10 @@ dkb.token.consumer_secret=
 # or equal than current_time then current token will be used for the request.
 # Default value is 60 seconds.
 # dkb.token.seconds_before_token_expiration=60
+
+# Santander Configuration
```
Comment (spetrychenko) [inline xs2a-adapter-service-api/src/main/resources/adapter.config.properties]: please update release notes

Comment (n0rdy) [reply xs2a-adapter-service-api/src/main/resources/adapter.config.properties]: My bad, done, thanks!

File: adapters/santander-adapter/pom.xml (hunk 1 above)
Comment (n0rdy) [reply adapters/santander-adapter/pom.xml]: Updated, please, take a look

**Pull request merged**